vividus-tests/src/main/resources/properties/suite/system/mobile_app/ios/suite.properties

```diff
@@ -1 +1 @@
-bdd.batch-1.meta-filters=groovy: !targetPlatform || targetPlatform.contains('ios')
+bdd.batch-1.meta-filters=groovy: (!targetPlatform || targetPlatform.contains('ios')) && !healthCheck
```

Comment:
I believe it's better to create a new story, like: HealthCheckMobileAppSteps.story